addons/spreadsheet_account/models/spreadsheet_mixin.py

```diff
@@ -0,0 +1,27 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from datetime import date
+
+from odoo import models
+from odoo.tools import date_utils
+
+
+class SpreadsheetMixin(models.AbstractModel):
+    _name = "spreadsheet.mixin"
+    _inherit = ["spreadsheet.mixin"]
+
+    def _get_spreadsheet_metadata(self, access_token=None):
+        metadata = super()._get_spreadsheet_metadata(access_token=access_token)
+        company = self.env["res.company"].browse(self.env.company.id)
+        start, end = date_utils.get_fiscal_year(
+            date.today(),
+            day=company.fiscalyear_last_day,
+            month=int(company.fiscalyear_last_month),
+        )
+
+        return {
+            **metadata,
+            "current_fiscal_year_start": start.isoformat(),
+            "current_fiscal_year_end": end.isoformat(),
```

Comment:
the format used widely by the server and expected by the webclient is not exactly ISO... (it's the default format used by postgres, I think)
```suggestion
            "current_fiscal_year_start": Date.to_string(start),
            "current_fiscal_year_end": Date.to_string(end),
```
We are doing our own fully custom thing, but let's be consistent anyway.
This might require to adapt the client-side to parse the values with `deserializeDate`